readme.md

```diff
@@ -542,6 +523,26 @@ Default: `SIGTERM`
 
 Signal value to be used when the spawned process will be killed.
 
+#### signal
+
+Type: `object (AbortSignal)`
+
+You can abort the spawned process using [`AbortController`](https://developer.mozilla.org/en-US/docs/Web/API/AbortController).
+
+```js
```

Comment:
Could you please move this example to the `Cancelling a spawned process` section above? For consistency, since all examples are currently grouped in the "Usage" and "Tips" sections. Thanks!